### ISSUE TITLE
uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library

### ISSUE THREAD

**Issue by Mani-99yards** [stripe/stripe-react-native#1760]
> Task :stripe_stripe-react-native:processDebugAndroidTestManifest FAILED
/Users/node_modules/@stripe/stripe-react-native/android/build/intermediates/tmp/manifest/androidTest/debug/tempFile1ProcessTestManifest7010220358703027879.xml:5:5-74 Error:
	uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [com.facebook.react:react-android:0.75.4] /Users/manikumargouni/.gradle/caches/8.9/transforms/1246311b17a3d477485f82624d8e0407/transformed/react-android-0.75.4-debug/AndroidManifest.xml as the library might be using APIs not available in 21
	Suggestion: use a compatible library with a minSdk of at most 21,
		or increase this project's minSdk version to at least 23,
		or use tools:overrideLibrary="com.facebook.react" to force usage (may lead to runtime failures)

See https://developer.android.com/r/studio-ui/build/manifest-merger for more information about the manifest merger.

**Comment by Mani-99yards** [comment]
I recently migrated from 0.73.4 to 0.75.4, I am getting above issue while building in android

**Comment by AlfaizTenup** [comment]
Same issue here.

**Comment by GouniManikumar12** [comment]
> Same issue here.
Are you able to find any fix?

**Comment by Joshandrews43** [comment]
Also experiencing this issue

**Comment by devonrmcclure** [comment]
I was experiencing this issue with as well. I solved it by using https://docs.expo.dev/versions/latest/sdk/build-properties/ and then adding 

```json
[
  "expo-build-properties",
  {
    "android": {
      "minSdkVersion": 26
    }
  }
],
```

to my `app.json` file under `plugins`. Set the version to what the error says is the higher min version.

**Comment by Asscher** [comment]
I'm also experiencing this issue with React Native 0.75.5 and @stripe/stripe-react-native 0.42.0.
Has anyone found a definitive fix?

**Comment by porter-stripe** [comment]
We recently released a new version with support for new architecture. Are you still seeing the issue on the latest version?

**Comment by Asscher** [comment]
> We recently released a new version with support for new architecture. Are you still seeing the issue on the latest version?

Yes also had that issue on 0.43.0 plus i had to update kotlin version, which gave issues for other older dependencies

**Comment by umerfarok** [comment]
@devonrmcclure Thanks! that worked for me
